build.gradle

```diff
@@ -6,17 +6,31 @@ group 'org.example'
 version '1.0-SNAPSHOT'
 
 repositories {
-    mavenCentral()
+    //mavenCentral()
```

Comment:
We need to keep `mavenCentral()` here. Not sure why there is an Artifacotry instance here.

Reply:
I just wanted to resolve the dependencies from Artifactory  ( preferably the  one I was going to run this functional test on). But I agree we can just use mavenCentral() as  well.